tests/e2e/cases/ci.sh

```diff
@@ -69,28 +74,22 @@ function test::ci::_scale_cluster() {
   log::info "cluster scaled to $1"
 }
 
-function test::ci::_chaos() {
+function test::ci::_random_crash() {
   size=$1
-  iters=$2
   max_kill=$((size / 2))
```

Comment:
The max_kill should be size/2+1

Reply:
That is a majority number? Here is max kill